image-rs/src/nydus/utils.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023. Nydus Developers. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use nydus_api::BuildTimeInfo;
+use oci_distribution::manifest;
+
+pub fn is_nydus_data_layer(desc: &manifest::OciDescriptor) -> bool {
+    desc.annotations
+        .as_ref()
+        .unwrap()
```

Comment:
It would be better to avoid `unwrap()` when possible:)
```
desc.annotations
        .as_ref()
        .map(|v| v.contains_key(super::NYDUS_META_LAYER))
        .unwrap_or_default()
```